ember_debug/route-debug.js

```diff
@@ -43,7 +43,9 @@ export default EmberObject.extend(PortMixin, {
   }),
 
   routeTree: computed('router', function() {
-    let routeNames = this.get('router.router.recognizer.names');
+    const router = this.get('router');
+    const routerLib = router._routerMicrolib || router.router;
+    let routeNames = routerLib.get('recognizer.names');
```

Comment:
@bgentry - The remaining CI failure is from this change.  `routerLib.get` is not a function, but since it is not relying on Ember's object model (e.g. no CP's) we can just do `routerLib.recognizer.names` directly.